src/metpy/plots/declarative.py

```diff
@@ -1262,3 +1266,248 @@ def _build(self):
             u.values[wind_slice], v.values[wind_slice],
             color=self.color, pivot=self.pivot, length=self.barblength,
             transform=transform)
+
+
+@exporter.export
+class PlotObs(HasTraits):
+    """The highest level class related to plotting observed surface and upperair data.
+
+    This class collects all common methods no matter whether plotting a upper-level or
+    surface data using station plots.
+
+    List of Traits:
+      * level
+      * time
+      * fields
+      * locations (optional)
+      * time_range (optional)
+      * formatters (optional)
+      * colors (optional)
+      * symbol_mapper (optional)
+      * vector_field (optional)
+      * vector_field_color (optional)
+      * reduce_points (optional)
+    """
+
+    parent = Instance(Panel)
+    _need_redraw = Bool(default_value=True)
+
+    level = Union([Int(allow_none=True), Instance(units.Quantity)])
+    level.__doc__ = """The level of the field to be plotted.
+
+    This is a value with units to choose the desired plot level. For example, selecting the
+    850-hPa level, set this parameter to ``850 * units.hPa``. For surface data, parameter
+    must be set to 'None'.
```

Comment:
Probably should be `` `None` `` to make clear it's the Python object.

Reply:
Done